src/solver/optimisation/hourly_csr_problem.h

```diff
@@ -50,14 +44,18 @@ class HOURLY_CSR_PROBLEM
 
     int hourInWeekTriggeredCsr;
     double belowThisThresholdSetToZero;
-    PROBLEME_HEBDO* problemeHebdo;
-    HOURLY_CSR_PROBLEM(int hourInWeek, PROBLEME_HEBDO* pProblemeHebdo) :
-        hourInWeekTriggeredCsr(hourInWeek),
-        problemeHebdo(pProblemeHebdo)
+    PROBLEME_HEBDO* problemeHebdo_;
```

Comment:
I would remove the trailing "_" for consistency (or rename others)

Reply:
Conflicting reviews